aws-batch-pdal/batch/pipeline/script.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import os
+import boto3
+import aws_helper
+
+#this script
+print('Python script: ', sys.argv[0])
```

Comment:
Use structred logging, preferably JSON.

always include jobId and correlationId in the logs so they can be easily matched